test/ERC4626.t.sol

```diff
@@ -107,6 +109,109 @@ contract SparkVaultERC4626Test is SparkVaultTestBase {
         vm.stopPrank();
 
         skip(1 days);
+
+        totalAssets = vault.totalAssets();
+    }
+
+    function test_previewRedeem_revertsOverLiquidityBoundary() public {
+        // Deal value accrued to the vault
+        deal(address(asset), address(vault), totalAssets);
+
+        uint256 shares = vault.balanceOf(user1);
+
+        assertEq(shares, 1_000_000e6);
+
+        // Take 1 wei of liquidity from the vault
```

Comment:
Remove smallest unit of liquidity from the vault (apply everywhere)

Reply:
done